css-flexbox/css-flexbox_navigation/css/styles.css

```diff
@@ -45,3 +44,12 @@ ul {
   padding: 5px 14px;
   border-radius: 5px;
 }
+
+.navigation__menu {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
Great, you can also add `flex-wrap: wrap` and `gap: 10px` that in mobile versions menu items can move to a new line and everything would look more sympathetically. Something like that: 
![image](https://github.com/user-attachments/assets/5a2f98c4-47b8-48b6-80d4-d254dddeba24)

Reply:
Good point! I will change it. Thank you!